iothub/service/src/Microsoft.Azure.Devices.csproj

```diff
@@ -2,10 +2,13 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <LangVersion>9.0</LangVersion>
+    <EnableNETAnalyzers>True</EnableNETAnalyzers>
+    <AnalysisLevel>latest-recommended</AnalysisLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <RootNamespace>Microsoft.Azure.Devices</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NeutralLanguage>en-US</NeutralLanguage>
```

Comment:
Any reason why we're removing en-us from device and adding it to service? Not sure I understand

Reply:
I didn't remove it from device. I moved where it was, so it shows up consistently in the same place in the csproj.